library/worldgen/src/testmod/resources/quilt.mod.json

```
@@ -5,7 +5,7 @@
     "id": "quilt_worldgen_testmod",
     "version": "1.0.0",
     "metadata": {
-      "name": "Quilt Worldgen Test Mod",
+      "name": "Quilt Worldgen Test Mod", 
```

```suggestion
      "name": "Quilt Worldgen Test Mod",
```
This will put this file in parity to the upstream

---

I tried editing this from my phone, thought I got it